### ISSUE TITLE
NCBI gene data

### ISSUE THREAD

**Issue by rachellyne** [intermine/intermine#2070]
NCBI might have changed what they include in their gene file - or maybe we have changed what we load.  The All_Data.gene_info file has features under the type_of_gene column labeled "biological-region"  and "other" which are not genes but regulatory regions and other features.  These should probably not be loaded as genes in HumanMine.  They do not have location information - we might have used to specify that we only load data with genomic location.  We should look into this for next build.

**Comment by julie-sullivan** [comment]
These are the unique values for that column:

```
biological-region
miscRNA
ncRNA
other
protein-coding
pseudo
rRNA
scRNA
snoRNA
snRNA
tRNA
unknown
```

Should I load everything but `unknown` and `other`?

**Comment by julie-sullivan** [comment]
Do I also discard if that value is empty?

**Comment by rachellyne** [comment]
There are empty ones?  I guess discard then.

**Comment by julie-sullivan** [comment]
Fixed by c834db6388344c1587ffd3ff5b630803610983bd

Let's check this in the next build.

**Comment by danielabutano** [comment]
Close for now.
